walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Event.java

```diff
@@ -53,6 +54,13 @@ public Event(String title, TimeRange when, Collection<String> attendees) {
     this.attendees.addAll(attendees);
   }
 
+  public static final Comparator<Event> ORDER_BY_START = new Comparator<Event>() {
+    @Override
+    public int compare(Event a, Event b) {
+      return Integer.compare(a.when.start(), b.when.start());
+    }
+  };
```

Comment:
nit: this could be expressed in fewer lines by using a lambda expression

Reply:
Yeah that's true. But I just realized that it is not necessary anymore to sort the Events for my implementation. So I'll just go ahead and remove this Comparator since I'm not using it anymore. :)